### ISSUE TITLE
Update ruby version to 2.3

### ISSUE THREAD

**Issue by aivinog1** [yegor256/pdd#123]
After #122. Seem to be that the lowest ruby version of the project became 2.3. It needs to change it in the following files:

- [x] https://github.com/yegor256/pdd/blob/master/pdd.gemspec
- [x] https://github.com/yegor256/pdd/blob/master/.rubocop.yml
- [x] https://github.com/yegor256/pdd/blob/master/appveyor.yml

And maybe in other files.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@aivinog1[/z](https://www.0crat.com/u/aivinog1) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3T46CUJJ) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@0crat assign @aivinog1

**Comment by 0crat** [comment]
> @0crat assign @aivinog1  ([here](https://github.com/yegor256/pdd/issues/123#issuecomment-509944438))

@yegor256 The job [#123](https://github.com/yegor256/pdd/issues/123) assigned to @aivinog1[/z](https://www.0crat.com/u/aivinog1), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/6b82527a-9fe1-4b69-b197-f45671e5f087); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); @aivinog1[/z](https://www.0crat.com/u/aivinog1) is not a member of this project yet, but they can request to join, as [§1](http://www.zerocracy.com/policy.html#1) explains; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @aivinog1[/z](https://www.0crat.com/u/aivinog1)

**Comment by 0pdd** [comment]
@aivinog1 the puzzle [#134](https://github.com/yegor256/pdd/issues/134) is still not solved.

**Comment by aivinog1** [comment]
So, I think this issue is done. A new puzzle was created.

**Comment by aivinog1** [comment]
@yegor256 Why @0crat doesn't remove this issue from me? Where is my reputation points?

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/yegor256/pdd/issues/123#issuecomment-510424912); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Comment by yegor256** [comment]
@0crat status

**Comment by yegor256** [comment]
@aivinog1 there is a bug with Zerocrat, most probably. Report it.

**Comment by aivinog1** [comment]
@yegor256 Thanks!

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/pdd/issues/123#issuecomment-510437981))

@yegor256 This is what I know about this job in [C3T46CUJJ](https://www.0crat.com/p/C3T46CUJJ), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#123](https://github.com/yegor256/pdd/issues/123) is [in scope](http://www.zerocracy.com/policy.html#14) for 1days
  * The role is `DEV`
  * The job is assigned to @aivinog1[/z](https://www.0crat.com/u/aivinog1) for [1days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @aivinog1[/z](https://www.0crat.com/u/aivinog1): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3T46CUJJ?q=%7Bjob%3A%27gh%3Ayegor256%2Fpdd%23123%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
Job was finished in 26 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@dennis00010011b[/z](https://www.0crat.com/u/dennis00010011b) please review this job completed by @aivinog1[/z](https://www.0crat.com/u/aivinog1), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Job [#123](https://github.com/yegor256/pdd/issues/123) is not in the agenda of @aivinog1[/z](https://www.0crat.com/u/aivinog1), can't inspect

**Comment by 0crat** [comment]
The job [#123](https://github.com/yegor256/pdd/issues/123) is now out of scope
